tools/disasm/disasm.py

```diff
@@ -2099,6 +2108,8 @@ def find_late_rodata_start(rodata):
 files_spec = None
 with open("tools/disasm/files.txt", "r") as infile:
     files_spec = ast.literal_eval(infile.read())
+for segment in files_spec:
+    segment[1] = segment[1].replace("$(BASEROM)", str(baserom_segments_dir))
```

Comment:
Does doing this has any measurable impact on running the disassembler? Hopefully it doesn't but who knows

Reply:
I didn't measure but I bet it does not have any measurable impact